docs/DifferentiableFunctions.md

```diff
@@ -310,7 +310,7 @@ discussion](https://forums.swift.org/t/help-needed-with-retroactive-differentiab
 for more information.*
 
 ```swift
-import Darwin
+import Foundation
```

Comment:
@rxwei is this correct as well?

Reply:
This one is fine. `Foundation` transparently imports `Darwin`/`Glibc` based on the operating system.